spec/helpers/application_helper_spec.rb

```diff
@@ -2,20 +2,20 @@
 
 describe ApplicationHelper, type: :helper do
    
-   describe '#markdown' do
+  describe '#markdown' do
 
-     it 'renders HTML from a markdown String' do
-       text = '**Moomin!**'
-       expect(helper.markdown(text)).to eq("<p><strong>Moomin!</strong></p>\n")
-     end
+    it 'renders HTML from a markdown String' do
+      text = '**Moomin!**'
+      expect(helper.markdown(text)).to eq("<p><strong>Moomin!</strong></p>\n")
+    end
 
-     it 'raises a TypeError if input is not a String' do
-       invalid_input = 123
-       expect{ helper.markdown(invalid_input) }.to raise_error(TypeError)
+    it 'raises a TypeError if input is not a String' do
+      invalid_input = 123
+      expect{ helper.markdown(invalid_input) }.to raise_error(TypeError)
      end
-   end
+  end
 
-   describe '#inline_markdown' do
+  describe '#inline_markdown' do
```

Comment:
I didn't change any of the tests in lines 1-18 (above), I just fixed the indentation.